### ISSUE TITLE
ERROR: list index out of range

### ISSUE THREAD

**Issue by rpattabi** [ubuntu/ubuntu-make#88]
Ubuntu 14.04 LTS shows up this error while installing Android Studio.

```
[I Accept (a)/I don't accept (N)] a
Downloading and installing requirements                                                                                              |
100% |###############################################################################################################################|
Installing Android Studio
ERROR: A decompression to /opt/umake/android/android-studio failed: list index out of range                                          |
Traceback (most recent call last):
  File "/usr/lib/python3.4/concurrent/futures/thread.py", line 54, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/usr/lib/python3/dist-packages/umake/decompressor.py", line 111, in _decompress
    dir_path = glob(os.path.join(dest, dir))[0]
IndexError: list index out of range
ERROR: list index out of range
|####################################################################################################################################|
```

Not sure if this is because of /opt requiring sudo. If so, we shall get a prompt than error after long download.

**Comment by didrocks** [comment]
Thanks for reporting this bug.

Well, running with sudo definitively has issues (see bug #79), however, it shouldn't crash it that way.
Did you reproduce that regularly? See as well my workaround on the last message of the other bug report if you really want to install it in /opt.

**Comment by didrocks** [comment]
oh, also, maybe giving the exact command line you used (with the path input and so on) would as well be of a great help, thanks!

**Comment by rpattabi** [comment]
(Not at my m/c room right now) I remember the simplest command `umake android-studio`. No sudo.

It prompted me a path, for which I gave `/opt/umake/android/android-studio`. After that it took a while to download some stuff. During this time I was wondering why it didn't ask me that I need to sudo or something similar.

After the long wait I got the reported error. When I get a chance I will try again. But looks like, some basic checks are required in the script prior to big downloads, etc.
